attestation-service/src/token/ear_default_policy_cpu.rego

```diff
@@ -90,18 +96,20 @@ else := 3 if {
 
 ##### TDX
 executables := 3 if {
+    input.tdx
 	# Check the kernel, initrd, and cmdline (including dmverity parameters) measurements
-	input.tdx.quote.body.rtmr_1 in data.reference.rtmr_1
-	input.tdx.quote.body.rtmr_2 in data.reference.rtmr_2
+	input.tdx.quote.body.rtmr_1 in get_reference_value("rtmr_1")
```

Comment:
I think we can comment out all the usage of `get_reference_value`, just leave the determined `A=B` rules.